examples/jsonld/spdx-full-example.json

```diff
@@ -0,0 +1,234 @@
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#creationtool",
+    "type" : "Tool",
+    "name" : "tools-java",
+    "comment" : "Created by the FullSpdxV3Example.java utility in tools-java",
+    "externalRef" : [ {
+      "type" : "ExternalRef",
+      "locator" : [ "org.spdx:tools-java" ],
+      "externalRefType" : "mavenCentral"
+    } ],
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#garyagent",
+    "type" : "Agent",
+    "name" : "Gary O'Neall",
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "https://spdx.org/licenses/creatoragent/3_27_0",
+    "type" : "Agent",
+    "description" : "This object is created and maintained by the SPDX legal team (https://spdx.dev/engage/participate/legal-team/)",
+    "name" : "SPDX Legal Team",
+    "creationInfo" : "_:creationInfo_1"
+  }, {
+    "spdxId" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#docannotation",
+    "type" : "Annotation",
+    "subject" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#document",
+    "annotationType" : "other",
+    "statement" : "This document is for example purposes only",
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#bundle",
+    "type" : "Bundle",
+    "context" : "Custom Licenses",
+    "comment" : "This is just an example of a concrete Bundle class - the elements are not used elsewhere in the SPDX document",
+    "element" : [ "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#LicenseRef-CustomLicense1", "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#LicenseRef-CustomLicense2" ],
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#spdxorg",
+    "type" : "Organization",
+    "name" : "System Package Data Exchange (SPDX)",
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "https://spdx.github.io/spdx-spec/v3.0.1/examples/full-example-eaa46bdcfa20#softwareagent",
+    "type" : "SoftwareAgent",
+    "name" : "SPDX Spec Github CI",
+    "creationInfo" : "_:creationInfo_0"
+  }, {
+    "spdxId" : "http://spdx.org/licenses/Apache-2.0",
```

Comment:
In general, this would be an external reference to this object, not included verbatim here (the same with its creator).